valio-master/Valio/ItemTableViewCell.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ItemTableViewCell.swift
+//  Valio
+//
+//  Created by Sam Soffes on 6/6/14.
+//  Copyright (c) 2014 Sam Soffes. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+class ItemTableViewCell: UITableViewCell {
+	
+	// MARK: - Properties
+
+	lazy var titleLabel: UILabel = {
+		let label = UILabel()
+		label.setTranslatesAutoresizingMaskIntoConstraints(false)
+		label.font = UIFont(name: "Avenir", size: 12)
+		return label
+	}()
+	
+	lazy var timeLabel: UILabel = {
+		let label = UILabel()
+		label.setTranslatesAutoresizingMaskIntoConstraints(false)
+		label.font = UIFont(name: "Avenir-Light", size: 12)
+		label.textColor = UIColor(red: 0.631, green: 0.651, blue: 0.678, alpha: 1)
+		label.textAlignment = .Right
+		return label
+	}()
+	
+	lazy var lineView: UIView = {
+		let view = UIView()
+		view.setTranslatesAutoresizingMaskIntoConstraints(false)
+		view.backgroundColor = UIColor(red: 0.906, green: 0.914, blue: 0.918, alpha: 1)
+		return view
+	}()
+	
+	lazy var circleView: UIView = {
+		let view = UIView()
+		view.setTranslatesAutoresizingMaskIntoConstraints(false)
+		view.backgroundColor = UIColor.whiteColor()
+		view.layer.borderWidth = 1
+		view.layer.cornerRadius = 7
```

Comment:
As a rule it's better to do a `CAShapeLayer` with a rounded rect instead of corner radius, as it's more performant, especially if this is a `UITableViewCell`. 

Also a separate thing, all those views, should have an opaque `backgroundColor` set to minimize blending and again gain performance.